rls/src/main/java/io/grpc/rls/internal/LbPolicyConfiguration.java

```diff
@@ -344,4 +348,14 @@ public String toString() {
           .toString();
     }
   }
+
+  /** Exception thrown when attempting to parse child policy encountered parsing issue. */
+  public static final class InvalidChildPolicyConfigException extends Exception {
+
+    private static final long serialVersionUID = 0L;
+
+    public InvalidChildPolicyConfigException(String message) {
+      super(message);
```

Comment:
Maybe you can turn off writableStackTrace.

Reply:
done.